snow/engine/snowman/bootstrap/block_job.go

```diff
@@ -45,7 +44,6 @@ func (p *parser) Parse(ctx context.Context, blkBytes []byte) (queue.Job, error)
 }
 
 type blockJob struct {
-	parser                  *parser
```

Comment:
This isn't really part of this PR... but this was dead code and having it here made the test harder to make... win win removing it